test/jasm/Jasm.gmk

```diff
@@ -41,6 +41,13 @@ $(BUILDTESTDIR)/jasmreporting.ok: \
 	$(GREP) '\---jasm-stdout:' $(@:%.ok=%)/work/JasmUsageTest.jtr > /dev/null
 	$(GREP) '\---jasm-stderr:' $(@:%.ok=%)/work/JasmUsageTest.jtr > /dev/null
 	$(GREP) '\---jasm:' $(@:%.ok=%)/work/JasmUsageTest.jtr > /dev/null
+	# verify that the warning wasn't printed
+	$(GREP) 'Warning: Class file version not specified in file or by -cv parameter' \
```

Comment:
Would it make sense to (also) look for any other "Warning: ..." messages? 🤔

Reply:
I spoke to Christian about this and we decided to go ahead with the integration of this PR in the current form. Thank you Christian for the review.